snuba/consumers/consumer.py

```diff
@@ -566,7 +567,8 @@ def process_message(
             ),
         )
     except Exception as err:
-        logger.error(err, exc_info=True)
+        sentry_sdk.set_tag("invalid_message", "true")
```

Comment:
Isn't this going to be set to true for any further exception this process sends to sentry? 
In that case it would be wrong.

Reply:
Oops, thanks